src/components/Chat/BottomSheetGif.js

```diff
@@ -48,15 +41,34 @@ const BottomSheetGif = React.forwardRef((props, ref) => {
     setIsLoading(false);
   };
 
+  const searchGIF = React.useCallback(
+    debounce((search) => {
+      onLoad(search);
+    }, 500),
+    []
+  );
+
+  const handleChangeText = (text) => {
+    setSearchText(text);
+    if (text?.trim()?.length > 2) {
+      searchGIF(text);
+    }
+  };
+
+  const handleOnClearText = () => {
+    setSearchText('');
+    searchGIF('');
+  };
+
   React.useEffect(() => {
-    onLoad();
+    onLoad('');
   }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useEffect` hook has been updated to call `onLoad('')` instead of `onLoad()`. This change aligns with the updated signature of the `onLoad` function and ensures that the initial load does not include a search query.
```diff
- React.useEffect(() => {
-   onLoad();
- }, []);
+ React.useEffect(() => {
+   onLoad('');
+ }, []);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->